libraries/stdlib/src/kotlin/collections/AbstractIterator.kt

```diff
@@ -13,8 +13,8 @@ private enum class State {
 }
 
 /**
- * A base class to simplify implementing iterators so that implementations only have to implement [[computeNext()]]
- * to implement the iterator, calling [[done()]] when the iteration is complete.
+ * A base class to simplify implementing iterators so that implementations only have to implement [computeNext]
```

Comment:
I'm not sure that it's worth documenting. The code is simple and the overhead is small, and if the user cares about such low-level details, they'll know not to use the class.